csrankings-j.csv

```diff
@@ -132,6 +132,7 @@ Jaehyuk Huh,KAIST,http://calab.kaist.ac.kr/~jhuh,NC4hxqEAAAAJ
 Jaeil Kim,Kyungpook National University,https://sites.google.com/view/ipalab,L8cMLmEAAAAJ
 Jaejin Lee,Seoul National University,http://aces.snu.ac.kr/~jlee,6JaKru0AAAAJ
 Jaejun Yoo,UNIST,https://sites.google.com/view/jaejunyoo,7NBlQw4AAAAJ
+Jaekyun Moon,KAIST,http://comstolab.kaist.ac.kr/,dfH3YpkAAAAJ

```

Comment:
EE, not CS. Please re-open with evidence of how this faculty member meets the inclusion criteria spelled out in the pull request form.